src/LiveComponent/doc/index.rst

```diff
@@ -625,10 +625,33 @@ persisted entities, which dehydrate to an ``id``).
 Hydration, DTO's & the Serializer
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you try to use a ``LiveProp`` for some unsupported type (e.g.a DTO object),
-it will fail. A best practice is to use simple data.
+.. versionadded:: 2.11
+
+    Dto are now a supported type for LiveComponent
+
+To use a ``LiveProp`` with DTO simply type properly your property
```

Comment:
You can also use a DTO (i.e. data transfer object / any simple class) with `LiveProp` as long as the property has the correct type::